File: CSharp_ChildrenCompetitionSockets/CSharp_ChildrenCompetitionSockets/model/TestAgeCategory.cs
﻿using System;

namespace CSharp_ChildrenCompetitionGUI.model
{
    [Serializable]
    public class TestAgeCategory : Entity<int>
    {
        public int minAge { get; }
        public int maxAge { get; }

        public TestAgeCategory(int minAge, int maxAge)
        {
            this.minAge = minAge;
            this.maxAge = maxAge;
        }
    }
}

File: Java_ChildrenCompetitionSockets/CompetitionModel/src/main/java/competition/model/TestAgeCategory.java
package competition.model;

public class TestAgeCategory extends Entity<Integer> {
    private int minAge;
    private int maxAge;

    public TestAgeCategory(){

    }

    public TestAgeCategory(int minAge, int maxAge) {
        this.minAge = minAge;
        this.maxAge = maxAge;
    }

    public int getMinAge() {
        return minAge;
    }

    public int getMaxAge() {
        return maxAge;
    }
}
